<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="DESKTOP-KIS4N2E" failures="1" tests="2" name="Default test(failed)" time="0.104" errors="0" timestamp="2022-03-22T10:43:15 IST">
  <testcase classname="failedxml.failedxmlstudy" name="TC2" time="0.067"/>
  <testcase classname="failedxml.failedxmlstudy" name="TC4" time="0.002">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at failedxml.failedxmlstudy.TC4(failedxmlstudy.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- TC4 -->
  <testcase classname="failedxml.failedxmlstudy" name="TC1" time="0.0">
    <ignored/>
  </testcase> <!-- TC1 -->
  <testcase classname="failedxml.failedxmlstudy" name="TC3" time="0.0">
    <ignored/>
  </testcase> <!-- TC3 -->
  <testcase classname="failedxml.failedxmlstudy" name="TC5" time="0.0">
    <ignored/>
  </testcase> <!-- TC5 -->
</testsuite> <!-- Default test(failed) -->
